### DISCUSSION TITLE
What happened to decoration:multisample_edges?

### DISCUSSION THREAD

**Discussion by DenverEllis** [hyprwm/Hyprland#3536] [Q&A] [answered]
I have this variable in one of my configs. I've tried searching where this field could have migrated to or been replaced by, but I can't find anything on where it went, or if it is still supported. It is not in the wiki either. This is not a breaking issue, but I definitely feel like I am overlooking something. 

Can someone help me remove the lovely red bar from the top of my screen? :)

**Comment by vaxerski** [comment, marked as answer]
it was announced on discord and will be announced when a new release happens. It was removed.

**Reply by DenverEllis** [reply]
Cheers! Thanks for the speedy reply! Suppose that means I can just remove it from my config without repercussion, but I'll check the discord first :)

Thanks for creating this awesome project, and for continuing to maintain it so diligently. I really appreciate your work and general awesomeness!

**Reply by vaxerski** [reply]
you can indeed just remove it. A better method is implemented by default.

Thanks :)
